src/Compilers/VisualBasic/Portable/Binding/Binder_Expressions.vb

```diff
@@ -1514,6 +1517,17 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Return arrayInitialization
         End Function
 
+        Private Function ReclassifyTupleLiteralExpression(
+                                                         tupleLiteral As BoundTupleLiteral,
+                                                         diagnostics As DiagnosticBag
+                                                         ) As BoundExpression
```

Comment:
Inconsistent signature formatting. See formatting used by ReclassifyUnboundLambdaExpression and BindAssignmentTarget below.